apps/verticalhome/index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="style/css/app.css">
     <link rel="stylesheet" href="style/css/pinned_apps.css">
     <link rel="stylesheet" href="shared/elements/gaia_grid_rs/style/external.css">
+    <link rel="stylesheet" href="shared/style/sk_key_utils.css">
```

Comment:
This file doesn't exist. Similarly, for shared/js/sk_key_utils.js . For these files need to add pr with 'soft key' implementation. So, now with these lines build is failed.